admin/modules/cookies/api/class-cookies-api.php

```
@@ -159,12 +159,20 @@ public function get_item_object( $item = null ) {
 	 *
+	 * Merges the admin-only script fields back in so REST callers (which run
+	 * through the 'edit' context check in prepare_item_for_response) still
+	 * receive opt_in_script / opt_out_script. Other consumers of
+	 * Cookie::get_prepared_data() — such as the category controller — do not
+	 * see those fields, preventing accidental exposure of raw JS.
+	 *
 	 * @since  3.0.0
 	 * @param  Cookie $object Cookie instance.
 	 * @return array
 	 */
 	protected function get_formatted_item_data( $object ) {
```

PHPStan (2.1.54)

[error] Return type (array) of method FazCookie\Admin\Modules\Cookies\Api\Cookies_API::get_formatted_item_data() should be compatible with return type (WP_Error) of method FazCookie\Admin\Modules\Cookies\Api\API_Controller::get_formatted_item_data()


(method.childReturnType)